core/src/lib/parse_conf.cc

```diff
@@ -73,13 +73,15 @@
 #define MAX_PATH 1024
 #endif
 
-void PrintMessage(void* sock, const char* fmt, ...)
+bool PrintMessage(void* sock, const char* fmt, ...)
 {
   va_list arg_ptr;
 
   va_start(arg_ptr, fmt);
   vfprintf(stdout, fmt, arg_ptr);
   va_end(arg_ptr);
+
+  return true;
```

Comment:
This is always true. As a matter of fact does not change code behaviour anywhere. 

You could just easily revert all changes from bool to void and remove the part of the false code branch above because it is dead code. (Please show me an other part in case I was wrong.)